samples/bluetooth/peripheral/src/main.c

```diff
@@ -378,9 +425,13 @@ int main(void)
 	while (1) {
 		k_sleep(K_SECONDS(1));
 
-		/* Current Time Service updates only when time is changed */
-		cts_notify();
-
+		/* Current time update notification example
+		 * For testing purpose we do update it periodically to send notifications
```

Comment:
"For testing purposes, we send a manual update notification every second." The rest of the sentence is fine.